spot-client/src/spot-remote/app-state/actions.js

```diff
@@ -145,14 +146,29 @@ export function connectToSpotTV(joinCode, shareMode) {
          * @returns {Promise}
          */
         function onDisconnect(error) {
-            logger.error('On Spot Remote disconnect', { error });
+            const usingPermanentPairingCode = getPermanentPairingCode(getState()) === joinCode;
```

Comment:
Does the permanent code not change? It's always the join code value even after code refreshes?

Reply:
Yes, with the current code(that is pending refactor, that's why it's confusing) it does not change, because it's not really used as long as the backend has access/refresh token stored and valid.

Reply:
Also the permanent code does not refresh really. Maybe you're thinking about long live remote pairing code.